locales/en-US/app.ftl

```diff
@@ -12,9 +12,17 @@ footerLinkPrivacy = Privacy
 footerLinkTerms = Terms
 footerLinkLegal = Legal
 footerLinkFeedback = Give Feedback
-footerLinkAbout = About Test Pilot
-
-## Items in the menu.
+footerLinkAboutHeader = About
+footerLinkAboutUs = About us
```

Comment:
Should this be "About Mozilla"? I would expect *us* = Test Pilot

Reply:
us in this case is mozilla, but It could be test pilot I suppose